README.md

```diff
@@ -1 +1,31 @@
-# Gym-Landing-Page
+#Life2Fit (Gym-Landing-Page)
```

Comment:
If this file is about adding a `README`, you should create another issue to handle this.
Please, remove this file from this pull request and create another issue for another pull request.